src/lib/viz/interactivity/viewport-features/ViewportFeaturesGenerator.ts

```diff
@@ -102,7 +102,7 @@ export class ViewportFeaturesGenerator {
   }
 
   private getSelectedTiles(): ViewportTile[] {
-    if (!this.deckLayer) {
+    if (!this.deckLayer || !this.deckLayer.state) {
```

Comment:
what's the purpouse of this check?

Reply:
In some examples this method threw me an error because `this.deckLayer` has value but its `state` was null.

If nobody has got that error likely it would be a random error...